application/controllers/Users.php

```diff
@@ -0,0 +1,95 @@
+<?php
+class Users extends CI_Controller {
+
+
+	public function __construct()
+    {
+        parent::__construct();	
+
+        // This line will load user model to this controller
+        $this->load->model('users_model');
+		$this->load->helper('url');
+    }
+
+    public function index()
+    {
+        // Use method get_users of user model to get users list
+        $data["users"] = $this->users_model->get_users();
+		$data["page_title"] = "List of users";
+
+        $this->load->view('header', $data); 
+
+        // assign users list to view
+        $this->load->view('users/index', $data);
+        $this->load->view('footer');
+    }
+	public function create()
+	{
+		$this->load->helper('form');
+		$this->load->library('form_validation');
+		$this->load->helper('url');
```

Comment:
We should put these to the constructor instead because they are called repeatedly